cves/CVE-2015-8213.yml

```diff
@@ -173,17 +173,17 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: Public post describing cve and fix on django development site.
+  name: utils
```

Comment:
General rule I've picked up from dev work, util folder/files are the work of the devil. Everything, by definition, has utility, so it isn't super descriptive. Maybe look back at the spec and try to find something better.